pyproject.toml

```diff
@@ -4,13 +4,9 @@ requires = [
   "numpy==1.13.3; python_version=='3.6' and platform_machine != 'aarch64' and platform_machine != 'arm64'",
   "numpy==1.17.0; python_version=='3.7' and platform_machine != 'aarch64' and platform_machine != 'arm64'",
   "numpy==1.17.5; python_version=='3.8' and platform_machine != 'aarch64' and platform_machine != 'arm64'",
-  "numpy==1.19.3; python_version<='3.9' and sys_platform == 'linux' and platform_machine == 'aarch64'",
```

Comment:
We need explicit `<2` in `setup.py` for Python 3.8 and earlier in case of possible expanding of numpy2 releases.

Reply:
It would clarify the actual state, but the current version works as well since numpy itself has `python_requires>=3.9`, so numpy 2.0 and later cannot be installed with older Python versions.

The added constraints here look good to me.